### DISCUSSION TITLE
What is the current best full stack with react?

### DISCUSSION THREAD

**Discussion by Zulaxy** [community/community#65693] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was wondering is MERN still the best stack with react or should I pick different db than MongoDB?

**Comment by Wickedlolz** [comment, marked as answer]
MERN (MongoDB, Express.js, React, Node.js) stack was a popular choice for building full-stack web applications, particularly for developers who prefer JavaScript throughout their tech stack. However, the choice of technology stack can vary based on your specific project requirements and the evolving landscape of web development.
